### PR TITLE
Feature - Support Swift 4 and Xcode 9.2

### PR DESCRIPTION
**Pull request by ghost**
- Change addNotificationBlock to observe
- Update RealmSwift and ObjectMapper
- Update Readme

### CODE REVIEW

**Review by ghost — APPROVED** [review]
@at-suho @at-daonguyen 
LGTM
@at-daonguyen 
Please check once again

File: Podfile [hunk 1]
```diff
@@ -4,13 +4,13 @@ use_frameworks!
 platform :ios, '8.0'
 
 def shared_pods
-    pod 'RealmSwift', '~> 2.2'
-    pod 'ObjectMapper', '~> 2.2'
+    pod 'RealmSwift', '~> 3.1.1'
```
Comment (at-daonguyen) [inline Podfile]: RealmSwift should be ~> 3.0

File: Podfile [hunk 2]
```diff
@@ -4,13 +4,13 @@ use_frameworks!
...
+    pod 'ObjectMapper', '~> 3.1.0'
```
Comment (at-daonguyen) [inline Podfile]: ObjectMapper should be ~> 3.0

**Review by at-daonguyen — CHANGES_REQUESTED** [review]
@at-suho Please check my comments.

**Comment by ghost** [comment]
@at-daonguyen Please check this pull request again, thank you.

**Review by at-daonguyen — APPROVED** [review]
LGTM!

**Pull request merged**